### ISSUE TITLE
can't join the network

### ISSUE THREAD

**Issue by shrisha108** [gravitl/netmaker#746]
Hello,
Looking for some help since I can't find any solution on-line. I just perform quick-install to ubuntu 20.04 VM on ORACLE Cloud and everything seems went smoothly. So I created network, made access keys and When trying to add first node I'm getting this error:
```
2022/02/12 20:32:23 [netclient] error installing: rpc error: code = Unknown desc = error: node traffic key is nil
2022/02/12 20:32:23 Failed to authenticate: rpc error: code = Unauthenticated desc = Something went wrong with Auto Login: rpc error: code = Unknown desc = no result found
2022/02/12 20:32:23 [netclient] removed private network locally
2022/02/12 20:32:23 [netclient] removed systemd remnants if any existed
```
System is Fedora 35. What could be a cause of this? I checked few times, token is correct. Actually I just copying installation line from Dashboard. Is this could be a bug?
Thank you.

**Comment by afeiszli** [comment]
Hi, issue is likely that you are missing the version flag. Please try the same command but include VERSION=v0.10.0. We are currently hotfixing this.

**Comment by afeiszli** [comment]
fixed in v0.11.0 and fixed in v0.10.0
